.gitattributes

```diff
@@ -3,3 +3,4 @@ test/**/*.xlsx filter=lfs diff=lfs merge=lfs -text
 public/**/*.xlsx filter=lfs diff=lfs merge=lfs -text
 *.rb diff=ruby
 *.rake diff=ruby
+resources/**/* filter=lfs diff=lfs merge=lfs -text
```

Comment:
I know I'm the one who said this folder should be called `resources` 🤦, but now I'm thinking `user-guides` would be a better, more descriptive name.

Also, let's add a sentence somewhere early in the README that points to this folder.